scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -523,148 +523,190 @@ global.base_stats = { //tempory stats subject to change by anyone that wishes to
 			base_group : "astartes",
 			traits:["ancient","slow_and_purposeful","lead_example","zealous_faith",choose("still_standing","beast_slayer","lone_survivor")]
 	},			
+	/* TODO - check and tweak if necessary
+	"chapter_servitor":{
+			title : "Chapter Servitor",
+			strength:[36,4],
+			constitution:[36,3],
+			weapon_skill : [30,2,"max"],
+			ballistic_skill : [30,2,"max"],
+			dexterity:[36,3],
+			intelligence:0,
+			wisdom:0,
+			charisma :[10,1],
+			religion : "imperial_cult",
+			piety : [28,3],
+			luck : 10,
+			technology :[30,3],
+			skills: {weapons:{"Bolter":1, "Chainsword":1, "Combat Knife":1, "Bolt Pistol":1}},
+			start_gear:{"armour":"power_armour", "wep1":"Combat Knife", "gear":"Servo-arm"}, TODO - tweak and check if correct
+			base_group : "astartes",
+			traits:["lobotomized"],
+	},
+	*/
+	// TODO - add more hirelings on the imperial side...
 	"skitarii":{
 			title : "Skitarii",
-			strength:20,
-			constitution:28,
-			weapon_skill : [20,5],
-			ballistic_skill : [20,5],			
-			dexterity:25,
-			intelligence:25,
-			wisdom:10,
-			charisma :5,
+			strength : [12,1], // I don't think skitarii are particularly strong
+			constitution : [15,1],
+			weapon_skill : [12,1],
+			ballistic_skill : [20,1],			
+			dexterity : [25,1],
+			intelligence : [25,1],
+			wisdom : [10,1], // Hm, no, very unwise...
+			charisma : [5,1], // Talking in binary does not help to make many friends
 			religion : "cult_mechanicus",
-			piety : 20,
-			technology :30,
-			luck :5,
-			skills: {weapons:{"hellgun":1,}},	
-			start_gear:{"armour":"skitarii_armour", "wep1":"hellgun"},
+			piety : [20,1],
+			luck : 10, // I don't see the point to make them less lucky than space marines
+			technology : [30,1],
+			skills: {weapons:{"Hellgun":1, "Combat Knife":1}},
+			start_gear:{"armour":"Skitarii Armour", "wep1":"Hellgun", "wep2":"Combat Knife"},
 			base_group : "skitarii",
 	},
```

Comment:
So, you've basically nerfed skitarii into the ground?

Reply:
2.5 * 15 = 37.5. So, roughly 35 hp per skitarii.
Most hirelings are nerfed, as 10, 10, 10, 10 is or at least was the baseline for human stats.
Skitarii are stronger than guardsmen in this case.
I don't think it is that bad - their ranged stats are pretty alright.

Reply:
Though, I should probably buff their armour a bit, right now it has lke 5 AC only. Should I go ahead with it?

Reply:
Although, I should probably lower the hands requirement for hellguns to just 1 as well, if I go at it.

Reply:
Ok, their armour has slightly been buffed now, to compensate.
But the armour still probably needs more buffs.

Reply:
My problem with these nerfs, is that you can justify them through lore perspective, but gameplay-vice you just made the unit worse, kept the price the same, creating a disbalance.

Reply:
Fair.
I have seen where the price for hirelings is located, assuming it works, I can lower it.
Should I also lower the price, then?

Reply:
Addressed by lowering prices and disposition requirements. Marking it as resolved, then.